messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/deployment/injection/AbstractJMSContext.java

```diff
@@ -65,11 +70,21 @@ private JMSContext createContext(JMSInfo info, ConnectionFactory connectionFacto
         return context;
     }
 
+    /**
+     * If context is closed by transaction listener, it should not be closed again in PostDestroy method.
```

Comment:
I suppose you mean in `@PreDestroy`, not `@PostDestroy`?

Reply:
@jmesnil  Yes, my bad, I'll fix it in a moment